fr/core-libraries/collections.rst

```diff
@@ -36,7 +36,7 @@ Collection serait::
     });
 
 Le :php:trait:`~Cake\\Collection\\CollectionTrait` vous permet également
-d'intégrer des fonctionnalité semblabes aux collections pour tout objet
+d'intégrer des fonctionnalité semblables aux collections pour tout objet
```

Comment:
fonctionnalités
Collections (with a capital C) maybe ? If it refers more to the object than the "concept"